README.md

```diff
@@ -317,6 +318,25 @@ Component structure:
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Component event names
+
+Vue.js provides events scoped to the component it emits and they are named as the developers desires. `~Make this paragraph better`
```

Comment:
Providing more links seems a good idea in general to clarify the concepts behind the styles. I would add here as well a [link](https://vuejs.org/v2/guide/events.html) from which text you can also get ideas to improve the text.